### PR TITLE
Order model

### PR DESCRIPTION
**Pull request by Alexsandr-Yablunovskyi**

### CODE REVIEW

File: src/main/java/yablunovskyi/bookstore/service/impl/OrderServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    private User findUser() {
+        String email = SecurityContextHolder.getContext().getAuthentication().getName();
+        return userRepository.findByEmail(email).orElseThrow(()
+                -> new EntityNotFoundException("Can't find user by email %s".formatted(email)));
+    }
+    
+    private Order createNewOrder(User user, CreateOrderRequestDto requestDto) {
+        Order order = new Order();
+        order.setOrderDate(LocalDateTime.now());
+        order.setUser(user);
```
Comment (OlehReex) [inline src/main/java/yablunovskyi/bookstore/service/impl/OrderServiceImpl.java:100]: You can create mapper to implement this logic

**Review by OlehReex — APPROVED** [review]
Looks like you should separate Order and ShoppingCart PR`s

**Review by Elena-Bruyako — CHANGES_REQUESTED** [review]
There are changes from Shopping Cart HW, please fix it

**Review by Elena-Bruyako — APPROVED** [review]
Good job!

**Pull request merged**